calibration/DOFSTK/CGINS-DOFSTK-03370__20200814.csv

```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3370,CC_frequency_offset,-8.192113e+002,source_file DOFST-K_SBE43F_SN_3370_Calibration_2020-08-14.zip > 3370.cal and SBE 43F O3370 SOCAdjusted 17Aug20.pdf
+43-3370,CC_oxygen_signal_slope,2.8370e-004,use adjusted Soc
```

Comment:
All values look good.  Need to be sure this slope ('adjusted Soc') value is in the instrument.